module.exports = async (ctx, next) => {
    try {
        await next();
    } catch (err) {
        console.error(err);
        ctx.status = 500;
        ctx.body = 'Internal Server Error';
    }
};